scripts/index-genenetwork

```diff
@@ -549,7 +525,11 @@ def is_data_modified(xapian_directory: str,
             ])
         # Return a zero exit status code when the data has changed;
         # otherwise exit with a 1 exit status code.
-        if (db.get_metadata("generif-checksum").decode() == hash_generif_graph(sparql_uri) and
+        generif = pathlib.Path("/var/lib/data/generif-metadata.ttl")
+        generif_checksum = "-1"
+        if generif.exists():
```

Comment:
I think we can move this check to the `hash_generif_graph` and return "-1" if it doesn't exist. This prevents duplicating this logic

Reply:
See: ff96efc26c5f